Sources/NIOPosix/Bootstrap.swift

```diff
@@ -993,6 +1015,33 @@ public final class ClientBootstrap: NIOClientTCPBootstrapProtocol {
         }
     }
 
+    #if canImport(Darwin) || os(Linux)
+    /// Specify the VSOCK address to connect to for the `Channel`.
+    ///
+    /// - parameters:
+    ///     - address: The VSOCK address to connect to.
+    /// - returns: An `EventLoopFuture<Channel>` for when the `Channel` is connected.
+    public func connect(to address: VsockAddress) -> EventLoopFuture<Channel> {
```

Comment:
this is public API so shouldn't be `#if os(...)`'d. In the body you can `#if os(...)` and just return an unsupported operation error.

Reply:
Yeah same reason as above